source/extensions/filters/http/jwt_authn/filter.cc

```diff
@@ -16,6 +16,7 @@ namespace HttpFilters {
 namespace JwtAuthn {
 
 namespace {
+const std::string INVALID_TOKEN_ERROR_STRING = "invalid_token";
```

Comment:
prefer not to use std::string in static,  can we use string_view,  or char[]

Reply:
Swapped to string_view